### ISSUE TITLE
Hot code-reloading deploys error out during upgrades

### ISSUE THREAD

**Issue by HashNuke** [HashNuke/ansible-elixir-stack#4]
An issue related to exrm or relx causes creating releases to error out. I've reported the issue here - https://github.com/bitwalker/exrm/issues/167

Been updating details from debugging sessions and identified the problem.

**Comment by manukall** [comment]
i wonder if the solution to this is to simply run the app from another directory, as the exrm doc suggests. would you accept a PR that adds a `/srv/app/{{ app_name}}` directory and deploys the app there?

**Comment by HashNuke** [comment]
@manukall Yes. that would be wonderful. Right now the ansible scripts just start the release in the place it's built. The setup needs to follow the exrm docs instead and start the release after copying it elsewhere. By deploying to another directory (like `/home/{{ deployer_user} }/srv/{{ app_name }}` is a good place because all apps are under the non-root deployer user.

**Comment by HashNuke** [comment]
Haven't looked into this yet. I'll do it by this weekend.

**Comment by vysakh0** [comment]
@HashNuke  I've followed the `docs`. And I get an error that some how I couldn't get past :disappointed: 

During this step

```
TASK: [HashNuke.elixir-stack | upgrade app] *********************************** 
<> ESTABLISH CONNECTION FOR USER: deployer
```

I get this error

```
stderr: fs: -c: line 0: syntax error near unexpected token `newline'
fs: -c: line 0: `rel/myapp/bin/myapp upgrade ==>'
```

I tried `upgrading` the exrm release manually and it works.  Seems to be a problem with shell script? It would be great you could help :+1:

**Comment by HashNuke** [comment]
Hot-code reloading is broken AFAIK.
I'll have to update it this weekend.

**Comment by pinx** [comment]
I would like to experiment with hot upgrade, so a fix would be welcome.

**Comment by HashNuke** [comment]
@pinx being worked upon in another branch. Here's the status https://github.com/HashNuke/ansible-elixir-stack/issues/23. Hopefully will be over by this weekend.
